docs/Architecture.md

```diff
@@ -1,42 +1,55 @@
+Maestro is composed of Management API, Rooms API, Operation Execution Worker, Runtime Watcher Worker, and Metrics Reporter Worker. Each one with its own responsibilities and is broken enough in a way to avoid mixing the execution process. Each module was thought to avoid parallel problems and to give the client more visibility about which **Operations** are being executed and their respective status.
 
-The proposed model is very similar to what continuous integration services do. You have a definition of what has to be done, and there is an implementation that takes it and executes it. This describes the overall model of this version. We call it "Operations".
 
+![architecture IMAGE](./diagrams/Architecture.jpg)
 
-## Architecture
+### Maestro modules
 
-Besides the new model described above, this release presents a completely different architecture and code organization.
+> ⚠ Note: Maestro currently only supports Kubernetes as Game Rooms runtime system. So Workers interact with them.
 
-```
-                        │
-Current components (v9) │ Next components (v10)
-                        │
-┌──────┐ ┌───┐          │ ┌─────────────────┐    ┌─────────┐
-│Worker│ │API│          │ │Operations Worker│    │Rooms API│
-└──────┘ └───┘          │ └─────────────────┘    └─────────┘
-                        │
-                        │ ┌──────────────┐ ┌───────────────┐
-                        │ │Management API│ │Runtime Watcher│
-                        │ └──────────────┘ └───────────────┘
-```
+#### Management API
 
--   Management API (Public API): This is the API that manages the scheduler and rooms. It is the component the users (usually game engineers) use while interacting with Maestro, making updates, and fetching scheduler information;
--   Rooms API (Internal API): API used by the game rooms to report their status to Maestro and communicate any extra information that may apply;
--   Operations executor worker: Worker that takes queued operations and processes them;
--   Runtime watcher: Watches any state change that happens on the Runtime and takes the necessary action (like updating some Maestro state or triggering an operation);
+Management API is the module responsible for receiving user requests. It accepts gRPC and HTTP requests. It could be used to fetch a **Scheduler** status or else to execute a change in them. There is some different kind of routes. In the **Scheduler** creation, it simply adds a new entry in the Postgres (and the worker posteriorly creates a respective queue to it). In the fetch **Scheduler** routes, it fetches from Postgres the **Scheduler** information and in the Redis the **Operations**/game rooms information. And in the change routes, it creates **Operations** and enqueues them in the respective **Scheduler** queue in the Redis. Finally, fetch **Operation**/game rooms routes it fetches its information from Redis.
```

Comment:
I think you don't have to describe the operations... Maybe simply describing it's the API used to interact with the scheduler as mentioned at Scheduler.md#how-to-operate would be enough